addons/mail/models/ir_attachment.py

```diff
@@ -44,7 +44,7 @@ def _delete_and_notify(self):
                 })
         self.unlink()
 
-    def _attachment_format(self, commands=False):
+    def _attachment_format(self, commands=False, format_token=False):
```

Comment:
The parameter name is quite confusing, it doesn't actually format anything. Maybe `include_token` would be clearer?